src/nixpacks/nix/pkg.rs

```diff
@@ -22,7 +23,7 @@ impl Pkg {
         match &self.overrides {
             Some(overrides) => {
                 let override_string = overrides
-                    .iter()
+                    .par_iter()
                     .map(|(name, value)| format!("{} = {};", name, value))
```

Comment:
I don't think we should be using rayon for small cases like this. `overrides` is at most going to be length 3 and all the map is doing is formating a string. I think the overhead of rayon will actually be slower.

Reply:
Ah, thought overrides was going to be longer then that.